Makefile

```diff
@@ -17,5 +17,6 @@ push-hail-ci-build-image: hail-ci-build-image
 
 deploy:
 	rm -f dist/*
-	python setup.py bdist_wheel
+	python2 setup.py bdist_wheel
```

Comment:
why not also deploy the source distribution for python2? Is that not available for python2's `setup.py`?

Reply:
or, alternatively, why publish the source distribution for python3?

Reply:
I was kind of under the impression that the source distribution for either of them was the same, and basically just the contents of the `cloudtools` folder or maybe this repository or something. So I could publish both, or neither, but I thought it would be good to have the source to fall back on, and it didn't seem worth publishing twice, but maybe I don't understand how that works?